service/templates/cloudformation/guest/auto_scaling_group.yaml

```diff
@@ -17,9 +17,6 @@
         - Key: Name
           Value: {{ .ClusterID }}-{{ .ASGType }}
           PropagateAtLaunch: true
-        - Key: KubernetesCluster
-          Value: {{ .ClusterID }}
-          PropagateAtLaunch: true
```

Comment:
Old tags are removed.

Reply:
Nice! I've just realized that we need an additional cluster tag in the public subnet, otherwise the services of type LoadBalancer won't work. This change makes that the subnets will be tagged too, is that correct?